bundles/core/src/test/resources/searchnlister/exporter-searchnlister-v1.json

```diff
@@ -0,0 +1,10 @@
+
+{
+    "id"                    : "searchnlister-ff2359bd7e",
+    "title"                 : "Sample Title",
+    "layout"                : "Custom",
+    "advancedSearchDisabled": true,
+    "textSearchDisabled"    : true,
+    "resultLimit"           : 4,
+    ":type"                 : "core/fd/components/formsportal/searchnlister/v1/searchnlister"
```

Comment:
We should probably add all assetPath here, probably a default value with empty string

Reply:
Should we leave assetPath detail upto the implementation?
It's not used during the rendering (sightly files) and doesn't seem to be useful if we're providing results? 

Also please review the current model json at https://github.com/adobe/aem-core-forms-components/pull/36/files#diff-cdfaeb2de1a9700eb5c370150dde6dff21ac7f126183d19c3bd3b47bdba0f32a